appveyor.yml

```diff
@@ -1,9 +1,5 @@
 version: "{build}"
 
-branches:
-  only:
-    - master
```

Comment:
Is there any way to reconfigure this so that Appveyor only builds on master or on `*-releases` branches?  So that commits on other branches that don't have a PR yet don't get built.